src/main/common/filter.c

```diff
@@ -302,3 +302,33 @@ float firFilterDenoiseUpdate(firFilterDenoise_t *filter, float input)
         return filter->movingSum / ++filter->filledCount + 1;
     }
 }
+
+// Fast two-state Kalman
+void fastKalmanInit(fastKalman_t *filter, float q, float r, float p, float initialValue)
```

Comment:
`initialValue` not required, since it is only ever set to zero. That should save a few bytes of ROM.